server/src/database/seeding/seed-data.ts

```diff
@@ -127,6 +127,46 @@ export function seedAssets(): CreateAssetDto[] {
       quantity: 12,
       imgUrls: [assetImg],
     },
+    {
+      title: 'Thirteen',
+      description: 'lucky lucky lucky',
+      type: AssetType.REQUEST,
+      condition: Condition.NONE,
+      quantity: 13,
+      imgUrls: [assetImg],
```

Comment:
Nice job! looks great. Could you change the image link to another public small image like any image from https://unsplash.com/s/photos/small-size-image? I know this isn't part of the ticket, but the link for `imageUrls` has been broken for some time

Reply:
@esteban-gs jesse and i both see that image on the FE so maybe you need to clear your browser cache??
otherwise, what do you mean by the img being broken?

Reply:
I think this may be related to how we load the image, then. Out of all the images in the app, my ad blocker stops this one image specifically from loading. But this shouldn't keep this PR from being merged.